demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -572,7 +572,9 @@ def _collect_layouts_dependencies(pack_layouts: list,
         for layout in pack_layouts:
             layout_data = next(iter(layout.values()))
             layout_dependencies = set()
-            if layout_data.get('group') == 'indicator' or layout_data.get('kind') == 'indicatorsDetails':
+            if layout_data.get('definitionId'):
+                layout_type = "Generic"
+            elif layout_data.get('group') == 'indicator' or layout_data.get('kind') == 'indicatorsDetails':
```

Comment:
```suggestion

            elif layout_data.get('group') == 'indicator' or layout_data.get('kind') == 'indicatorsDetails':
```

Also the later variables names `related_incident_or_indicator_types` and `related_incident_or_indicator_fields` need to change as well.

Reply:
Done